dashboard/views/signup.ejs

```diff
@@ -49,13 +49,18 @@
                   <input type="text" data-i18n="[placeholder]email" class="form-control" name="email"
                     value="<% if(typeof user == 'object' && user.email){ %><%= user.email %><% } %>">
                 </div>
-                <div class="form-group col-md-12">
+                <div class="form-group col-md-12" id="password">
                   <label for="password" data-i18n="password"></label>
                   <input type="password" data-i18n="[placeholder]new-password" class="form-control" name="password"
-                    autocomplete="new-password">
+                    autocomplete="new-password" id="password-box">
                 </div>
+                <div class="form-group col-md-12">
+                  <label for="confirm-password" data-i18n="confirm-password"></label>
+                  <input type="password" data-i18n="[placeholder]confirm-new-password" class="form-control" name="confirm-password"
+                    autocomplete="confirm-new-password" id="confirm-password">
+                  </div>
```

Comment:
No need to add 2 new locales. The same result can be obtained by using `confirm-password` only